packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,450 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { EXPLORER_URL } from '../../config';
+import { useFungibleTokensIncludingNEAR } from '../../hooks/fungibleTokensIncludingNEAR';
+import { checkAccountAvailable } from '../../redux/actions/account';
+import { clearLocalAlert, showCustomAlert } from '../../redux/actions/status';
+import NonFungibleTokens, { NFT_TRANSFER_GAS } from '../../services/NonFungibleTokens';
+import isMobile from '../../utils/isMobile';
+import { formatNearAmount } from '../common/balance/helpers';
+import FormButton from '../common/FormButton';
+import Modal from '../common/modal/Modal';
+import ModalFooter from '../common/modal/ModalFooter';
+import SafeTranslate from '../SafeTranslate';
+import ReceiverInputWithLabel from '../send/components/ReceiverInputWithLabel';
+import AvatarSuccessIcon from '../svg/AvatarSuccessIcon';
+import EstimatedFees from '../transfer/EstimatedFees';
+
+
+const StyledContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 40px 0 0 0;
+
+    #nft-transfer-modal {
+        width: 400px;
```

Comment:
Can we avoid defining a width here since we've already added a `modalSize` as a prop for the `Modal` component?